apisix/plugins/openid-connect.lua

```diff
@@ -554,7 +555,7 @@ local function validate_claims_in_oidcauth_response(resp, conf)
 end
 
 function _M.rewrite(plugin_conf, ctx)
-    local conf = core.table.clone(plugin_conf)
+    local conf = fetch_secrets(plugin_conf, true, plugin_conf, "")
```

Comment:
Please still clone the table. 
While it is true that `fetch_secrets` does imply this fact, it cannot be inferred from the name and is not a feature of any mandatory guarantee.
As soon as someone modifies `fetch_secrets` in the future and doesn't clone there anymore, something will go wrong with your code.

Therefore, it's best to explicitly copy this table here anyway to ensure that, no matter how much external conditions change, the logic of this plugin doesn't break.

Reply:
I previously referred to the implementation of other plugin codes:
- https://github.com/apache/apisix/blob/master/apisix/plugins/authz-keycloak.lua#L767
- https://github.com/apache/apisix/blob/master/apisix/plugins/limit-count.lua#L37

There are many other locations, so I think there is no problem with the current PR from the implementation point of view.

> As soon as someone modifies fetch_secrets in the future and doesn't clone there anymore, something will go wrong with your code.

If this happens, many plugin codes will need to be adjusted. If my understanding is wrong, please correct me.